### ISSUE TITLE
Archery Gameplay Overhaul XPMSE Compatibility Issue AND Pretty Combat Animations

### ISSUE THREAD

**Issue by TwistedModding** [ForgottenGlory/Living-Skyrim-2#144]
**If you are reporting a crash to desktop, please attach your NET Script Framework crash log. This can be found in MO2's Overwrite folder.

If possible, please also attach a copy of your most recent save before the issue occurred.**

**LS Version**
2.0.0 beta 3

**Describe the bug**
Shooting a bow in 3rd person takes longer than in first person when using "frostfall quiver" in xpmse. Instead of pre-drawing the arrow (like in first person) you will instead just hold the arrow in your other hand making it so you have to completely redraw the arrow spending extra time. 
*** UPDATE - There is also a conflict with pretty combat animations causing the animation to take a long time on female characters only even with the frostfalls quiver disabled.

**To Reproduce**
Take any bow and arrow put frostfall quiver on in xpmse styles menu.

**Expected behavior**
Pre-drawn arrow like first person and default quiver position.

**Screenshots**
N/A

**Additional context**
Here is a possible fix i found just by a quick google search https://www.reddit.com/r/skyrimmods/comments/b13cje/archery_gameplay_overhaul_xpmse_compatibility_help/

**Comment by TwistedModding** [comment]
An alternative fix that could work is use AGOs belt fastened quiver support under the files and use the mod "Belt Fastened Quivers" instead of XPMSE's "frostfall quiver"

**Comment by ForgottenGlory** [comment]
Should be fixed Beta 4, followup on this.

**Comment by TwistedModding** [comment]
> Should be fixed Beta 4, followup on this.

One issue is fixed but it brought up three more.
------------------------------------------------------
New Issue 1 - When using "Frostfall quiver" on males in XPMSE it will now predraw like its suppose to but it will draw from the "default" quiver location instead of the frostfall making it so you're drawing arrows from thin air. 
New Issue 2 - For some reason when on a male character while equipping the bow, there will also be a duplicate of the bow on your characters back. 
New Issue 3 - I can't get frostfall quiver to even enable on female character.

------------------------------------------------------
Old Issues - Females still have all mentioned issues (will repost right here)
- Conflict between AGO and Pretty Combat Animations causing the "predraw" not to work. 
*NOTE* Before I updated I was playing around with the list trying to fix the issue between AGO and PCA and was somewhat successful. The only thing is that the bow animations from PCA will no longer work (i'm pretty sure there is no way around this and the fix is a little jerry rigged.) 

Copy all bow related animations from AGO in the path: D:\LivingSkyrim\mods\Archery Gameplay Overhaul SE\meshes\actors\character\animations to D:\LivingSkyrim\mods\Pretty Combat Animations\meshes\actors\character\animations

This will get rid of PCA's animations for archery and instead use AGOs (You might be able to find a different fix but this is what I did for testing purposes.) 

I believe i covered everything I just tested.

**Comment by TwistedModding** [comment]
OK. So an update on this again...

I have found this mod: https://www.nexusmods.com/skyrimspecialedition/mods/34889 which seems to fix the compatibilty issues between AGO and XPMSE completely getting rid of all problems with the male character. (YES I know additions are not supported, I will be reinstalling the list to get rid of these 

Issues with the female character seem to still be there. The only solution I have found I posted in the   previous comment.

This is all the info I have and it should help you to fix it in the next beta. :)
